### DISCUSSION TITLE
Is it possible to sync showing tooltip of few charts?

### DISCUSSION THREAD

**Discussion by romanslonov** [f5/unovis#468] [Q&A]
Hello, looking for a way to sync a few charts similar to https://recharts.org/en-US/examples/SynchronizedLineChart But didn't find anything related. Is it possible?

**Comment by rokotyan** [comment]
Check out these examples:

https://unovis.dev/gallery/view?collection=Auxiliary%20Components&title=Grouped%20Bar%20with%20Brush%20and%20Interactive%20Legend

https://unovis.dev/gallery/view?collection=Auxiliary%20Components&title=Scatter%20Plot%20with%20Free%20Brush

**Reply by romanslonov** [reply]
Is it possible to sync tooltips actually? 
![CleanShot 2024-10-22 at 10 56 43@2x](https://github.com/user-attachments/assets/bdfc04f3-1611-4f6d-839b-803b82d072fd)

**Reply by rokotyan** [reply]
In theory — yes. The tooltip component has an API that you can use https://unovis.dev/docs/auxiliary/Tooltip#controls. But this won't be straightforward
